packages/sdk/src/types/Customers.ts

```diff
@@ -136,3 +137,55 @@ export type CustomersUpdateParams = CustomerRequest & IdOrSourceId
 export type CustomersUpdateResponse = CustomerObject | CustomerUnconfirmed
 
 export type CustomersUpdateConsentsBody = Record<string, boolean>
+
+// New types following guideline
+
+// 0-level types
+
+type CustomerBase = {
+	name?: string
+	description?: string
+	email?: string
+	phone?: string
+	birthdate?: string
+	metadata?: Record<string, any>
+}
+
+type CustomerAddress = {
+	address?: {
+		city?: string | null
+		state?: string | null
+		line_1?: string | null
+		line_2?: string | null
+		country?: string | null
+		postal_code?: string | null
+	} | null
+}
+
+export type CustomersUpdateInBulkRequestBody = CustomerBase &
+	CustomerAddress &
+	{
+		source_id: string
+	}[]
```

Comment:
As far as I see, the `CustomersUpdateInBulkRequestBody` type is not defined correctly. It should describe an array of objects like:

```ts
const someRequest:CustomersUpdateInBulkRequestBody = [
	{name: 'asdasd', source_id: '1234asd'}
]

```

but it does not. That's the reason for the TS error in tests:

![image](https://github.com/voucherifyio/voucherify-js-sdk/assets/35666945/b2bb9379-8197-4659-bafb-c0936c193783)


The type should look like 

```ts
export type CustomersUpdateInBulkRequestBody = (
		CustomerBase &
		CustomerAddress &
		{
			source_id: string
		}
	)[]
```

Reply:
Interestingly, my idea did not display this error and tests are passing 🫣